.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Report something that's not working correctly
+labels: kind/bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please fill the form below.  
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please provide a general description of the issue, including what you're trying to accomplish.
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Provide a link to a live example or an unambiguous set of steps to reproduce this bug.  
+        Please include any relevant code used.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Behavior
+      description: What happened instead?
+    validations:
+      required: true
+  - type: textarea
+    id: versions
+    attributes:
+      label: Versions used
+      description: Please provide the version of Pulumi and all relevant packages used.
```

Comment:
Just ask for the output of `pulumi about` from the project directory.

Reply:
ooooo nice, much easier!